src/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -888,7 +889,8 @@ def forward(
 
         if attention_mask is not None:
             # make sure padded tokens are not attended to
-            hidden_states[~attention_mask] = 0
+            expand_attention_mask = attention_mask.unsqueeze(-1).repeat(1, 1, hidden_states.shape[2])
```

Comment:
Sorry just one question - why do we use `repeat` here instead of https://pytorch.org/docs/stable/generated/torch.broadcast_to.html ? Broadcasting should save some memory no? 

Also could we please run all the Wav2Vec2 slow tests ones to make sure there is no unexpected numerical difference?